qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -226,7 +227,7 @@ def get_signals(self, schedule: Schedule) -> List[DiscreteSignal]:
             if sig.duration < max_duration:
                 sig.add_samples(
                     start_sample=sig.duration,
-                    samples=np.zeros(max_duration - sig.duration, dtype=complex),
+                    samples=unp.zeros(max_duration - sig.duration, dtype=complex),
```

Comment:
Very minor but I think this can just remain as `np`, as `max_duration - sig.duration` should always be a standard python `int`, so this will always just be dispatched to `numpy` anyway.